layouts/section/issues-baseof.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en">
+  {{ partial "head" . }}
+  <body data-spy="scroll" data-target="#issues-sidebar" data-offset="5" class="page-{{ .Section }}{{ if .IsHome }}home{{ end }}">
+    {{ partial "nav" . }}
+    {{ block "main" . }}{{ end }}
+
+    {{ $jquery := resources.Get "vendor/jquery-3.3.1.min.js" }}
```

Comment:
can we use the footer template here?
`{{ partial "footer" . }}`

Reply:
or rather, split the footer template into two, one for the content, one for the js resources